cachi2/core/package_managers/npm.py

```diff
@@ -502,6 +566,95 @@ def _get_npm_dependencies(
     return download_paths
 
 
+def replace_dependency(dependency_version: str) -> bool:
+    """Check if dependency must be updated in package(-lock).json.
+
+    package(-lock).json files require to replace dependency URLs for
+    empty string in git and https dependencies in V2+.
+    """
+    url = urlparse(dependency_version)
+    if url.scheme == "file":
+        return False
+    return not (url.scheme == "" and "/" not in dependency_version)
```

Comment:
Maybe it's just me, but I found this statement a little confusing. I believe it can be rewritten as:

```
return url.scheme != "" or "/" in dependency_version
```

Reply:
sure we can simplify